website/docs/r/cloudaccount_alibaba.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "dome9"
+page_title: "Check Point CloudGuard Dome9: dome9_cloudaccount_alibaba"
+sidebar_current: "docs-resource-dome9-cloudaccount-alibaba"
+description: |-
+  Onboard Alibaba cloud account
+---
+
+# dome9_cloudaccount_alibaba
+
+This resource is used to onboard Alibaba cloud accounts to Dome9. This is the first and pre-requisite step in order to apply  Dome9 features, such as compliance testing, on the account.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "dome9_cloudaccount_alibaba" "test" {
+  name                   = "NAME"
+  operation_mode         = "OPERATION MODE"
+  subscription_id        = "SUBSCRIPTION ID"
+  tenant_id              = "TENANT ID"
+  client_id              = "CLIENT ID"
+  client_password        = "CLIENT PASSWORD"
```

Comment:
this is not correct

Reply:
The brunch change did not keep the file up to date. Done